### PR TITLE
Products are filtered by price, stock, delivery, rating, and categories

### PR DESCRIPTION
**Pull request by parulgupta2001**

### CODE REVIEW

**Review by therajatg — APPROVED** [review]
code looks clean

File: my-app/src/frontend/components/navbar/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  return (
+    <nav class="navigation-bar">
+      <div className="upper-nav">
+        <span className="log-in">Login</span>
+        <span className="sign-up">Sign up</span>
+      </div>
+      <div className="main-nav">
+        <div className="logo-container">
+          <img
+            className="logo"
```
Comment (MaruthiPothuganti) [inline my-app/src/frontend/components/navbar/Navbar.js:17]: An image should always be provided with an `alt` tag to improve accessibility.

File: my-app/src/frontend/components/products/products.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+.main {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    column-gap: 3rem;
+    padding-left: 4rem;
+    padding-top:3rem;
```
Comment (MaruthiPothuganti) [inline my-app/src/frontend/components/products/products.css:7]: short hand properties can be used here. Flex-flow and padding property

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested some changes. Everything else looks good.

**Review by DivyanshSareen — APPROVED** [review]
good to merge

**Pull request merged**